### PR TITLE
Update readme for 1.0.0-beta2, remove format param reference

### PR DESCRIPTION
**Pull request by loosebazooka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'com.google.cloud.tools:endpoints-framework-gradle-plugin:1.0.0-beta'
+    classpath 'com.google.cloud.tools:endpoints-framework-gradle-plugin:1.0.0-beta2'
```
Comment (joaoandremartins) [inline README.md:21]: Is this going GA soon? If not, we could end up with a very large beta number, not sure if that's bad or not really.

File: README.md [hunk 2]
```diff
@@ -35,7 +35,6 @@ The plugin exposes the following server side goals
 * `endpointsDiscoveryDocs` - generate discovery documents
 
 The plugin exposes server side configuration through the `endpointsServer` extension
-* `format` - The output format of discovery documents [rest or rpc]
```
Comment (joaoandremartins) [inline README.md:38]: Is the support for this already removed in the codebase?

File: README.md (hunk 1 above)
Comment (loosebazooka) [reply README.md:21]: I think so, but this shouldn't change too much. Endpoints tools is on beta12

File: README.md (hunk 2 above)
Comment (loosebazooka) [reply README.md:38]: yeah it's gone

**Review by joaoandremartins — APPROVED** [review]

**Pull request merged**